test/unit/spec/EnrollWindowsHello_spec.js

```diff
@@ -187,6 +189,42 @@ function (Okta,
           });
         });
       });
+
+      itp('does not show error if webauthn.makeCredential fails with AbortError', function () {
+        return emulateWindows('AbortError')
+        .then(setup)
+        .then(function (test) {
+          test.setNextResponse([responseMfaEnrollActivateWebauthn, responseSuccess]);
+          test.form.submit();
+          return tick(test);
```

Comment:
We now have Expect statements - in this case, we should use Expect.waitForFormError and then do the checks in the next statement.